src/local_scheduler/local_scheduler_client.cc

```diff
@@ -56,8 +56,13 @@ void local_scheduler_disconnect_client(LocalSchedulerConnection *conn) {
   flatbuffers::FlatBufferBuilder fbb;
   auto message = ray::local_scheduler::protocol::CreateDisconnectClient(fbb);
   fbb.Finish(message);
-  write_message(conn->conn, static_cast<int64_t>(MessageType::DisconnectClient),
-                fbb.GetSize(), fbb.GetBufferPointer(), &conn->write_mutex);
+  if(conn->use_raylet) {
+    write_message(conn->conn, static_cast<int64_t>(MessageType::IntentionalDisconnectClient),
```

Comment:
(Moving conversation here): I was trying to `kill -9` an idle actor and obtain the task killed stderr response, but nothing was occurring. `kill -9` on a actor running a task will be fine. Shouldn't raise warnings at equal levels?